src/util-proto-name.c

```diff
@@ -196,3 +196,22 @@ bool SCProtoNameValid(uint16_t proto)
 {
     return (proto <= 255 && known_proto[proto] != NULL);
 }
+
+/**
+ * \brief   Function to return the protocol number for a named protocol.
+ *
+ * \param proto Protocol name.
+ * \retval ret On success returns the protocol number; else -1
+ */
+int SCGetProtoByName(char *protoname)
+{
+    if (protoname) {
+        for (int i = 0; i <= 255; i++) {
+            if (known_proto[i] &&
+                    0 == strncasecmp(known_proto[i], protoname, strlen(known_proto[i]))) {
```

Comment:
wondering if we should access any case mix here... e.g. tCp would work, and TcP... seems we should maybe stay close to what it was when using the `/etc/protocols`

Reply:
Ok ... I'll change this to not ignore case -- which is get `getprotobyname` was oing.